lib/src/install.rs

```diff
@@ -546,8 +548,11 @@ async fn initialize_ostree_root_from_self(
     // Another implementation: https://github.com/coreos/coreos-assembler/blob/3cd3307904593b3a131b81567b13a4d0b6fe7c90/src/create_disk.sh#L295
     crate::lsm::ensure_dir_labeled(rootfs_dir, "", Some("/".into()), 0o755.into(), sepolicy)?;
 
-    // TODO: make configurable?
-    let stateroot = STATEROOT_DEFAULT;
+    let stateroot = state
+        .config_opts
+        .stateroot
+        .as_deref()
+        .unwrap_or(STATEROOT_DEFAULT);
```

Comment:
This is just `let stateroot = state.config_opts.stateroot.as_str()` now right?

Reply:
Yeah of course